### PR TITLE
added articles to db

### PR DESCRIPTION
**Pull request by sephi-dev**

### CODE REVIEW

File: src/collections/Articles.ts [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import { CollectionConfig } from 'payload/types';
+
+// Example Collection - For reference only, this must be added to payload.config.ts to be used.
+const Categories: CollectionConfig = {
```
Comment (sephi-dev) [inline src/collections/Articles.ts]: You want to call it Products

File: src/collections/Articles.ts [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  slug: 'articles',
```
Comment (sephi-dev) [inline src/collections/Articles.ts]: the slug should be products

File: src/collections/Articles.ts [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+      name: 'image',
+      type: 'upload',
+      relationTo: 'media',
+      required: true,
+    },
+    {
+      name: 'quantity',
+      type: 'number',
+      required: true,
+    },
```
Comment (sephi-dev) [inline src/collections/Articles.ts]: I think the quantity should not be related to this collection.

File: src/payload-types.ts [hunk 4]
```diff
@@ -43,6 +43,13 @@ export interface Media {
   createdAt: string;
   updatedAt: string;
 }
+export interface Article {
+  id: string;
+  name: string;
+  image: Media;
+  price: number;
+  quantity: number;
+}
```
Comment (sephi-dev) [inline src/payload-types.ts]: Be careful, the file is auto generated

File: src/collections/Articles.ts (hunk 3 above)
Comment (zakariyarezok) [reply src/collections/Articles.ts]: Each product is created and then the quantity increases elsewhere?

Comment (sephi-dev) [reply src/collections/Articles.ts]: I think a stock collection is needed where we can handle the stock

**Comment by sephi-dev** [comment]
LGTM, you can merge this PR

**Pull request merged**